[CC-14622] ac doc documentation display discounted price on promotional product (#634)
spryker/spryker-docs: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/managing-discounts.md

## Editing a discount

To edit a discount, do the following:
1. On the **Overview of Discounts** page, click **Edit** in the **Actions** column.
2. Update the attributes and click **Save**.

{% info_block infoBox %}

> To find out more information about the attributes that you see on the **Edit Discount** page, in the Creating a voucher guides, see the [Reference information: Creating a voucher](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#reference-information-creating-a-voucher) section.

Reviewer: ```suggestion
For more information about the attributes on the **Edit Discount** page, see [Reference information: Creating a voucher](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#reference-information-creating-a-voucher).
```
